mountinfo/mounted_unix.go

```diff
@@ -38,7 +38,7 @@ func normalizePath(path string) (realPath string, err error) {
 		return "", err
 	}
 	if _, err := os.Stat(realPath); err != nil {
-		return "", err
+		return "", &os.PathError{Op: "stat", Path: realPath, Err: err}
```

Comment:
This is totally redundant; `os.Stat` already returns `PathError`.

https://go.dev/play/p/aaLjEvYTAfV